mesa/README.md

```diff
@@ -1 +1,4 @@
-# MESA
\ No newline at end of file
+# MESA
+
+# FRONT-END DEPENDENCIES:
```

Comment:
Change: `# FRONT-END DEPENDENCIES:`
To: `## FRONT-END DEPENDENCIES:`

Reply:
I will delete this line actually. I forgot that all of the dependencies I install go into "node_modules" which is then available to everyone when they merge the branch. So there's no need for this line.